hudi-common/src/main/java/org/apache/hudi/metrics/JmxMetricsReporter.java

```diff
@@ -72,9 +69,25 @@ public JmxMetricsReporter(HoodieMetricsConfig config, MetricRegistry registry) {
     }
   }
 
+  private boolean isServerCreated() {
+    return jmxReporterServer != null;
+  }
+
+  private void initializeJmxReporterServer(String host, int[] ports) {
+    for (int port : ports) {
+      try {
+        jmxReporterServer = createJmxReport(host, port);
+        LOG.info("Started JMX server on port " + port + ".");
+        break;
+      } catch (Exception e) {
```

Comment:
Should we catch specific exception type here?

Reply:
> Should we catch specific exception type here?

https://github.com/apache/hudi/pull/11226/commits/c7b402870d78e079662a5d810f7484e39dc20f83
Caller function catches all exceptions, then rethrow to HoodieException.
Thus, check causes and log messages in detail.

Thanks for comment